README.md

```diff
@@ -1,13 +1,81 @@
 # GridGopher
 
-A placeholder repository for Allegheny College automated grade transfer tool
+![License](https://img.shields.io/badge/license-MIT-blue?style=flat)
+![BuiltWith](https://img.shields.io/badge/Built%20With-Python-blue?style=flat&logo=python&logoColor=yellow)
+![Actions Status](https://github.com/noorbuchi/GridGopher/workflows/Lint%20and%20Test/badge.svg)
+[![codecov](https://codecov.io/gh/noorbuchi/GridGopher/branch/main/graph/badge.svg?token=02353FAN4W)](https://codecov.io/gh/noorbuchi/GridGopher)
+![stars](https://img.shields.io/github/stars/noorbuchi/GridGopher.svg)
 
-## Important Notes
+![GridGopherLogo](images/GopherLogo.png)
 
-- Values in the `.env` file must be surrounded by double quotation marks `"`
-  otherwise, newline character `\n` will cause issues.
+GridGopher is a plugin friendly tool that allows users to connect collected data
+from Google Sheets and GitHub issue trackers and pull requests. The tool
+provides the basic API and encourages users to utilize it in their applications.
 
-Expected Format:
+## Set Up and Installation
+
+**Installation using `pip` is currently not supported. However, it's planned for
+future releases**
+
+GridGopher uses Poetry to create a Python virtual environment and manage
+dependencies. For more information about Poetry, check out [the
+documentation](https://python-poetry.org/). To set up the tool for use, please
+follow the steps outlined below:
+
+**1- Install Poetry:**
+
+Install poetry using the steps outlined
+[here](https://python-poetry.org/docs/#installation). To verify that poetry was
+installed successfully run the following:
+
+```
+poetry -V
+```
+
+The expected output is the version of Poetry installed.
+
+**2- Install Python Dependencies:**
+
+Once poetry has been installed successfully, clone or download the repository
+and navigate to the root of the repository. Use the following command to install
+all the dependencies used by GridGopher:
+
+```
+poetry install
+```
+
+This command might take some time to finish running. Once it's completed,
+GridGopher is ready for use!
+
+## Running GridGopher
+
+### API Setup
+
+GridGopher requires authentication tokens for a Google Sheets API service
```

Comment:
Check the overall grammar of this sentence. It seems like the 's' on "accounts" should be deleted, but I am not entirely sure.